frontend/src/features/splReports/columns.tsx

```diff
@@ -53,6 +53,25 @@ export const columns: any[] = [
     width: spacing.medium,
     minWidth: 80,
   },
+  {
+    Header: 'Status',
```

Comment:
This is the `Risk` column, not the `Status`.

Reply:
I know, but in the excel export it's labelled as "status"

Reply:
The Excel report regrettably must match an existing format so that the clients other reports will continue to work.  Within PIMS we can label them correctly.